docs/installation.md

```diff
@@ -114,3 +114,16 @@ The following executables found installed, which are needed by unblob:
     yaffshiv                    ✓
     zstd                        ✓
 ```
+
+7. (Optional) Install it system wide.
+
+You can install unblob system wide in two operations:
+
+    1. Build the unblob distribution package:
+
+        cd unblob
+        poetry build
+
+    2. Install the unblob distribution package:
+
+        sudo python3 -m pip install dist/unblob-0.1.0.tar.gz
```

Comment:
Maybe put some placeholder to version instead of hardcoding it like:
```suggestion
        sudo python3 -m pip install dist/unblob-<version>.tar.gz
```